tests/unit/db/mysql/test_load_options.py

```diff
@@ -0,0 +1,305 @@
+import unittest
+from records_mover.records import DelimitedRecordsFormat
+from records_mover.db.mysql.load_options import MySqlLoadOptions, mysql_load_options
+
+
+class TestMySQLLoadOptions(unittest.TestCase):
+    maxDiff = None
+
+    # TODO: Should ESCAPED BY be...ESCAPED?
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker